### PR TITLE
Feature cs physics

### PR DESCRIPTION
**Pull request by Nexather**

### CODE REVIEW

**Review by starwarsamketner — APPROVED** [review]
hkgkgk

File: OrbitLearn/Assets/Scripts/Body.cs [hunk 1]
```diff
@@ -32,15 +28,11 @@ private void Awake()
 
     private void FixedUpdate()
     {
-        ApplyForce();
+        //ApplyForce();
```
Comment (Kacy-Luker) [inline OrbitLearn/Assets/Scripts/Body.cs:31]:
```suggestion
```

**Review by RyanDerr — APPROVED** [review]
This better be good

**Review by brendans1818 — APPROVED** [review]
LG2M

**Review by Kacy-Luker — APPROVED** [review]

**Pull request merged**